draft-ietf-rtcweb-jsep.xml

```diff
@@ -1243,6 +1243,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           and/or pending remote description according to the rules defined for
           Trickle ICE. Connectivity checks will be sent to the new candidate.</t>
 
+          <t>This method can also be used to provide an end-of-candidates
+          indication to the ICE Agent, as defined in
+          <xref target="I-D.ietf-ice-trickle"/>.</t>
```

Comment:
@pthatcherg I wasn't sure if JSEP was supposed to mention specifics about the API. But I'll change it and see if anyone objects.

@ekr Redundant because "candidate" from the above paragraph can include a null candidate? I assume the meaning of a null candidate needs to be defined here though, since it's not defined in Trickle ICE.

@juberti Done.